packages/compiler/init/init-template.ts

```diff
@@ -3,7 +3,7 @@ import { TypeSpecConfigJsonSchema } from "../config/config-schema.js";
 import { TypeSpecRawConfig } from "../config/types.js";
 
 export interface InitTemplateFile {
-  path: string;
+  path: string | boolean;
```

Comment:
hhm forgot it was an array of this object and not a map. This feels a little wrong to have that this way now

Reply:
wondering if we should have instead a new property `skip?: boolean` and I think in the future it could be used to conditionally exclude the file from parameters as well